src/Util/FileSystem.h

```diff
@@ -7,12 +7,7 @@
 #ifndef CARTA_BACKEND__UTIL_FILESYSTEM_H_
 #define CARTA_BACKEND__UTIL_FILESYSTEM_H_
 
-#ifdef _BOOST_FILESYSTEM_
```

Comment:
Also need to edit `CommonTestUtilities.h`

Reply:
Thanks!